style.css

```diff
@@ -90,3 +84,202 @@ h1 span {
   font-size: 25px;
   color: #505f79;
 }
+
+.work-section {
+  padding: 15px;
+  margin: 20px 20px 40px 20px;
```

Comment:
## [OPTIONAL] :clap: 

- [ ] I would suggest also working on the margin at the left side of the projects list. by using the `margin: 20px 0;` property in the `.work-section` class  check the picture below :arrow_lower_left: 

![Screenshot from 2021-11-27 15-55-07](https://user-images.githubusercontent.com/65068771/143684432-b375a00b-2181-463e-93b8-66e7f40c8b91.png)